bentoml/_internal/runner/container.py

```diff
@@ -172,7 +172,7 @@ def from_batch_payloads(  # pylint: disable=arguments-differ
         payloads: t.Sequence[Payload],
         batch_dim: int = 0,
         plasma_db: "ext.PlasmaClient" = Provide[DeploymentContainer.plasma_db],
-    ) -> t.Tuple["ext.NpNDArray", t.List[int]]:
+    ) -> t.Tuple["ext.NpNDArray", BatchIndicesType]:
```

Comment:
afaik, for params, the type annotation should be as general as possible. And for the return values, annotations should be as concrete as possible. Thus it's okay to keep this list[int] since it does return list[int]